src/main/java/Task.java

```diff
@@ -0,0 +1,84 @@
+public class Task {
+    private static final int MAX_NUMBER_OF_TASKS = 100;
+    protected String description;
+    private boolean isDone;
+    private static Task[] taskList = new Task[MAX_NUMBER_OF_TASKS];
+    private static int taskCount = 0;
+    private static String time;
+
+    // Task Constructor
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon(){
+        return(this.isDone ? "x" : " ");
+    }
+
+    public static void markAsDone(String i){
+        int index = Integer.parseInt(i);
+        Task t = taskList[index-1];
+        t.isDone = true;
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.print("  ");
+        t.printTaskInformation();
+        System.out.println("");
```

Comment:
I think this can be extracted out to form a method so others can have more clarity on what is going on